lib/cf/CloudControllerClient.js

```diff
@@ -227,19 +231,54 @@ class CloudControllerClient extends HttpClient {
   getSpaces(options) {
     return this.createSpaceStream(options).all();
   }
-  getOrg(space_guid, options) {
+
+  getSpace(space_guid, options) {
     return this.getResource(`/v2/spaces/${space_guid}`, options);
   }
+
   getOrgAndSpaceGuid(instance_guid) {
     return this.getServiceInstance(instance_guid)
-      .then(body => this.getOrg(body.entity.space_guid)
-        .then(org => ({
+      .then(body => this.getSpace(body.entity.space_guid)
+        .then(space => ({
           space_guid: body.entity.space_guid,
-          organization_guid: org.entity.organization_guid
+          organization_guid: space.entity.organization_guid
         }))
       );
   }
 
+  getServiceInstanceDetails(instance_guid) {
```

Comment:
Can this method also take additional `options` parameter? This param can have `space_guid` as well provided in input. By the time request reaches all the respective handlers, `space_guid` will be [populated](https://github.com/SAP/service-fabrik-broker/blob/master/lib/controllers/ServiceFabrikApiController.js#L122) in request object. So no need to fetch space guid via a seperate call and it can avoid one additional call to CC. 
Also, no need to get Service & Plan details from CC. It can be retrieved from `catalog.js`.